engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/interop/generic/GetMembersNode.java

```diff
@@ -15,14 +14,12 @@
 public class GetMembersNode extends Node {
   private @Child InteropLibrary library =
       InteropLibrary.getFactory().createDispatched(Constants.CacheSizes.BUILTIN_INTEROP_DISPATCH);
-  private final BranchProfile err = BranchProfile.create();
 
   Object execute(Object self, Object object) {
     try {
       return library.getMembers(object);
     } catch (UnsupportedMessageException e) {
-      err.enter();
-      throw new PanicException(e.getMessage(), this);
+      return Array.empty();
```

Comment:
This feels like a behaviour change - not sure of the implications.

Reply:
The `org.graalvm.polyglot.Value.getMemberKeys()` returns empty array when the message is not supported: https://github.com/oracle/graal/blob/60970aca5870c2a3065ec9212e6bb2a36f7665d2/truffle/src/com.oracle.truffle.polyglot/src/com/oracle/truffle/polyglot/PolyglotValueDispatch.java#L3267

`UnsupportedMessageException` is common and there is no reason to propagate it to user.